util/topsql/reporter/reporter.go

```diff
@@ -277,6 +277,8 @@ func (tsr *RemoteTopSQLReporter) doCollect(
 		delete(collectTarget, evict.Key)
 		normalizedSQLMap.Delete(string(evict.SQLDigest))
 		normalizedPlanMap.Delete(string(evict.PlanDigest))
+		tsr.sqlMapLength.Add(-1)
```

Comment:
It can be possible that SQLDigest is even not in the map.

Reply:
Nice catch, addressed.